readme.md

```diff
@@ -1 +1,14 @@
-To get this running, create a .env file with a KEY. This will be your authorization for uploading and deleting images
+# Auro Uploader
+
+## Setup:
+1. Install dependencies `yarn`
+2. Create a `.env` file containing these keys:
+```
+key=PRIVATE_KEY // This is used for Authorizing your upload requests.
+URL=YOUR_DOMAIN
+```
+3. Start the server. `node index.js`
+
+## Chatterino Uploader:
+![Chatterino Uploader Example](https://i.mmatt.link/2dTpe.png)
```

Comment:
Upload the file to GitHub that way in case the file on the server goes missing it is still accessible here